modules/osd-create-cluster-ccs.adoc

```diff
@@ -0,0 +1,221 @@
+endif::osd-on-gcp[]
+The key is used for encryption of persistent volumes in your cluster.
+.. Click *Next*.
+
+. On the *Default machine pool* page, select a *Compute node instance type* and a *Compute node count*. The number and types of nodes that are available depend on your {product-title} subscription. If you are using multiple availability zones, the compute node count is per zone.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type. The number and types of nodes available to you depend on your {product-title} subscription.
+====
+
+. Optional: Expand *Edit node labels* to add labels to your nodes. Click *Add label* to add more node labels.
+
+. In the *Cluster privacy* section, select *Public* or *Private* to use either public or private API endpoints and application routes for your cluster.
++
+[IMPORTANT]
+====
+If you are using private API endpoints, you cannot access your cluster until you update the network settings in your cloud provider account.
+====
+
+ifdef::osd-on-aws[]
+. Optional: To install the cluster in an existing AWS Virtual Private Cloud (VPC):
+.. Select *Install into an existing VPC*.
+.. Select *Use a PrivateLink* if you want to enable connections to the cluster by Red Hat Site Reliability Engineering (SRE) using only AWS PrivateLink endpoints instead of public endpoints. This option cannot be changed after a cluster is created.
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+. Optional: Select *Install into an existing VPC* to install the cluster in an existing GCP Virtual Private Cloud (VPC).
+endif::osd-on-gcp[]
+
+. Click *Next*.
+
+. If you opted to install the cluster in an existing 
+ifdef::osd-on-aws[]
+AWS 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+GCP 
+endif::osd-on-gcp[]
+VPC, provide your *Virtual Private Cloud (VPC) subnet settings*.
+ifdef::osd-on-aws[]
++
+[NOTE]
+====
+You must ensure that your VPC is configured with a public and a private subnet for each availability zone that you want the cluster installed into.
+====
+endif::osd-on-aws[]
+
+. In the *CIDR ranges* dialog, configure custom classless inter-domain routing (CIDR) ranges or use the defaults that are provided.
++
+[IMPORTANT]
+====
+CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
+====
+
+. On the *Cluster update strategy* page, configure your update preferences:
+.. Choose a cluster update method. By selecting *Individual updates*, you can schedule each update individually. If you select *Recurring updates*, your cluster is updated on your preferred day and start time, when updates are available. *Individual updates* is the default option.
++
+[NOTE]
+====
+You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
```

Comment:
ISG would like "lifecycle" to be one word but if it's already two words elsewhere in OSD docs then this is fine as is.